router-tests/modules/module_test.go

```diff
@@ -38,3 +39,103 @@ func TestModuleSetCustomHeader(t *testing.T) {
 		assert.JSONEq(t, res.Body, `{"data":{"employees":[{"id":1},{"id":2},{"id":3},{"id":4},{"id":5},{"id":7},{"id":8},{"id":10},{"id":11},{"id":12}]}}`)
 	})
 }
+
+func TestSortingModulesByPriority(t *testing.T) {
```

Comment:
Priority 0 won't change the previous behavior correct?

Reply:
Fixed

Reply:
Unit tests should be live in the router module.